### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by burkegg**

### CODE REVIEW

File: client/ReviewModule.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import $ from 'jquery';
 import TopBar from './TopBar.jsx';
 import Reviews from './Reviews.jsx';
+// import ReviewStyles from './styles/ReviewStyles';
```
Comment (alilthumuch) [inline client/ReviewModule.jsx]: Do you need this? if not, then can remove.

File: client/Reviews.jsx [hunk 2]
```diff
@@ -21,6 +26,7 @@ const UserWrapper = styled.span`
   width: 250px;
   padding-top: 0px;
   height: 300px;
+  margin-bottom:0px;
```
Comment (alilthumuch) [inline client/Reviews.jsx]: space after colon.

File: client/UserInfo.jsx [hunk 3]
```diff
@@ -29,6 +29,23 @@ const UserBottom = styled.div`
...
+const NameLink = styled.a`
+  text-decoration: none;
+  font-size: 1em;
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const UserTidbit = styled.div`
+
```
Comment (alilthumuch) [inline client/UserInfo.jsx]: Extra space

**Review by alilthumuch — APPROVED** [review]
Good stuff! Not too sure how to review the testing part since haven't gotten to it yet so you could have someone else look at it too.

File: client/ReviewModule.jsx (hunk 1 above)
Comment (burkegg) [reply client/ReviewModule.jsx]: Fixed.

File: client/Reviews.jsx (hunk 2 above)
Comment (burkegg) [reply client/Reviews.jsx]: Fixed.

File: client/UserInfo.jsx (hunk 3 above)
Comment (burkegg) [reply client/UserInfo.jsx]: Fixed.

**Pull request merged**